symfony/plugins/orangehrmCorePlugin/test/Dao/AccessFlowStateMachineDaoTest.php

```diff
@@ -261,15 +261,11 @@ public function testHandleUserRoleRename(): void
 
     protected function getWorkflowItemIdsForRole($flow, $role)
     {
-        $conn = $this->getEntityManager()->getConnection();
+        $conn = $this->getEntityManager()->getConnection()->getWrappedConnection();
```

Comment:
Are these changes related to the change?

Reply:
No. Upgrade doctrine 2.8 to 2.10, then had to refactor code.
This is the CSRF token related commit: https://github.com/orangehrm/orangehrm/pull/1002/commits/1b215cbd2f44b8d1be99102294e45c6bb41c12cd